int16 address
int8 command
int8 retries
int8 priority
---
int16 value
int8 status
